### DISCUSSION TITLE
Create Custom SCSS file

### DISCUSSION THREAD

**Discussion by suraj-nividata** [bootscore/bootscore#385] [Q&A]
How do I create a custom archive.scss file? I want it because archive.css will only be enqueued in some pages.

**Comment by justinkruit** [comment]
With our current system this isn't possible. The built in compiler will compile everything that's loaded into main.scss. Also, with the amount of css that's probably gonna be in that file, it'll be unnecessary to split that up. The first load of the website will have all the css and next page loads are from the browser cache so they load faster.
